pkg/internal/logging/utils.go

```diff
@@ -0,0 +1,46 @@
+package logging
+
+import (
+	"fmt"
+	"log/slog"
+	"os"
+
+	"github.com/4chain-ag/go-wallet-toolbox/pkg/defs"
+)
+
+const (
+	// ServiceKey is the key used to store the service name in the logger attrs.
+	ServiceKey = "service"
+)
+
+var strLevelToSlog = map[defs.LogLevel]slog.Level{
+	defs.LogLevelDebug: slog.LevelDebug,
+	defs.LogLevelInfo:  slog.LevelInfo,
+	defs.LogLevelWarn:  slog.LevelWarn,
+	defs.LogLevelError: slog.LevelError,
+}
+
+// Child returns a new logger with the given service name added to the logger attrs.
+func Child(logger *slog.Logger, serviceName string) *slog.Logger {
+	return DefaultIfNil(logger).With(
+		slog.String(ServiceKey, serviceName),
+	)
+}
+
+func Errorf(logger *slog.Logger, err error, format string, args ...any) {
+	logger.Error(fmt.Sprintf(format, args...), slog.String("error", err.Error()))
+}
```

Comment:
This is not what I meant. 
My suggestion was rather to provide wrapper over slog.String("error", err) 
something like:
```suggestion
func Error(err error) Args {
	return slog.String(ErrorKey, err.Error()))
}
```

Reply:
✅ Done